app/_locales/en/messages.json

```diff
@@ -2779,7 +2779,7 @@
     "message": "Recents"
   },
   "recipientAddressPlaceholder": {
-    "message": "Search, public address (0x), or ENS"
```

Comment:
For now, lets remove this change. This will trigger a visual change to the user that, if this ships separately from the UD work, will confuse our users.  This will also remove all the changes to the e2e spec files.